src/main/java/com/easysplit/ess/transactions/domain/contracts/TransactionsRepository.java

```diff
@@ -16,4 +16,20 @@ public interface TransactionsRepository {
      * @return transaction created
      */
     TransactionEntity createTransaction(TransactionEntity transactionEntity);
+
+    /**
+     *Gets a transaction by is guid
```

Comment:
Something minor, but we should add an blanck space after the *, this is not a blocker for this PR an can be fix in another one.